drivers/dma/Kconfig.gd32

```diff
@@ -0,0 +1,13 @@
+# Copyright (c) 2022 Tokita, Hiroshi <tokita.hiroshi@gmail.com>
+# SPDX-License-Identifier: Apache-2.0
+
+# Workaround for not being able to have commas in macro arguments
+DT_COMPAT_GD_GD32_DMA := gd,gd32-dma
+
+config DMA_GD32
+	bool "Gigadevice GD32 DMA driver"
+	depends on (SOC_FAMILY_GD32 || SOC_SERIES_GD32VF103)
+	select USE_GD32_DMA
+	default $(dt_compat_enabled,$(DT_COMPAT_GD_GD32_DMA))
```

Comment:
Hi @soburi, @cameled ,

Just a remind that Kconfig has been changing to a new approach, see #48122 as example.

Reply:
Use DT_HAS～ macro instead of dt_compat_enabled.